apps/condo/domains/analytics/access/TicketReportService.js

```diff
@@ -1,15 +0,0 @@
-const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
-
-const { queryOrganizationEmployeeFor } = require('@condo/domains/organization/utils/accessSchema')
-
-async function canReadTicketReportWidgetData ({ authentication: { item: user } }) {
```

Comment:
Why it’s deleted?

Reply:
Not used anywhere in code. (Found duplicate in ticket domain, which is actually used, so removed legacy code)